docker/php-fpm-pool.conf

```diff
@@ -6,11 +6,10 @@ listen = 9000
 
 request_terminate_timeout = 120s
 
-pm = dynamic
+pm = ondemand
```

Comment:
Minor reduction in RAM utilization on an idle system. Performance should be about the same overall.

Per https://ma.ttias.be/a-better-way-to-run-php-fpm/